spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -113,86 +102,119 @@ function ExecuteTx(sourcePort: string, destPort: string, destChannel: string, ms
 
-2. The controller chain emits an event signaling to open a new channel on this port given a connection 
-3. A relayer listening for `OnChannelOpenInit` events will begin the channel creation handshake
-4. During the `OnChanOpenTry` callback on the host chain an interchain account will be registered and a mapping of the interchain account address to the owner account address will be stored in state (this is used for authenticating transactions on the host chain at execution time)
-5. During the `OnChanOpenAck` & `OnChanOpenConfirm` callbacks on the controller & host chains respectively, the `active-channel` for this interchain account/owner pair, is set in state
+Due to how the mechanics of ICS-004 channel version negotiation operate the version passed into the host chain side (OnChanOpenTry) must match the counterparty version passed into the controller side (OnChanOpenAck) otherwise there will be a resulting error at the IBC protocol level. 
 
+Combined with the one channel per interchain account approach, this method of version negotiation allows us to pass the address of the interchain account back to the controller chain and create a mapping from controller port ID -> interchain account address during the `OnChanOpenAck` callback. As outlined in the [controlling flow](#Controlling-Flow), a controller chain will need to know the address of a registered interchain account in order to send transactions to the account on the host chain.
 
-**Controlling Flow**
 
-Once an interchain account is registered on the host chain a controller chain can begin sending instructions (messages) to control this account. 
+#### Version negotiation summary
 
-1. The controller chain calls `GetInterchainAccountAddressFromAck()` to get the address of the interchain account on the host chain and sets the address in state mapped to the respective portID. 
-2. The controller chain calls `TrySendTx` and passes message(s) that will be executed on the host side by the associated interchain account (determined by the source port identifer)
+| Initiator | Datagram         | Chain acted upon |  Version (Controller, Host) | Port ID (Controller, Host) |
+| --------- | ---------------- | ---------------- |  ---------------------- | ---------------------- |
+| Controller| ChanOpenInit     | Controller       |  (ics27-1, none)        | (ics27-1.0.0.<owner-id>, interchain-account)           |
+| Relayer   | ChanOpenTry      | Host             |  (ics27-1, ics27-1.<owner-id>)        | (ics27-1.0.0.<owner-id>, interchain-account)       |
```

Comment:
This table seems incorrect. `owner-id` sounds like the account address on the controller chain since that is the "owner" of the interchain account.

But from my understanding, the version set by host chain contains the address of the host chain that is being controlled by the owner address on the controller chain.

I think the address in version string should have a different name: `host-address`?

Reply:
great catch! this is indeed incorrect. <owner-address> here should be the interchain account address registered on the host chain.

Reply:
I updated the variable name to be `interchain-account-address` as I'm hesitant to add further terminology here. Open to discussion. 

I also removed the port-id column as I noticed that I wasn't using variable names for the connection identifiers and adding them in makes the column difficult to parse. I think it's fine without.

Reply:
The reason I replaced the `<>` with `{}` is simply because using `<>` inside a table makes the content invisible on github README for some reason.